content/docs/payment-providers/quickpay/1.0.0/sub-packages.yml

```diff
@@ -0,0 +1,2 @@
+- name: QuickPay V10
+  alias: v10
```

Comment:
This alias needs to match the folder name on disk / the URL so this is currently failing on the preview site and so is showing a 404.